pebblo/app/service/doc_helper.py

```diff
@@ -239,6 +239,30 @@ def _update_app_details(self, raw_data, ai_app_docs):
             loader["sourceFiles"] = new_source_files
         self.app_details["report_metadata"] = raw_data
 
+    @staticmethod
+    def _create_data_source_findings(data_source_findings):
+        for data in data_source_findings:
+            for snippet in data.get("snippets"):
+                entity_details = {}
+                topic_details = {}
+                if snippet.get("entityDetails"):
+                    if data["labelName"] in snippet.get("entityDetails").keys():
```

Comment:
Can we combine these nested if statements?

Reply:
sure